web_app/templates/login.html

```diff
@@ -122,5 +154,15 @@ <h2>Login</h2>
     ></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
     <script src="{{url_for('static', filename='script.js')}}"></script>
+
+    <script>
+      function handle_badge(badge) {
```

Comment:
When you include the live badge reader script, you just have to make sure you make a 'handle_badge' function which does the correct redirect in all of the associated templates :)